library/Fission/CLI/Command/App/Init.hs

```diff
@@ -133,4 +133,18 @@ parseOptions = do
     , short   'b'
     ]
 
-  return App.Init.Options {..}
+  optSubdomain <- strOption $ mconcat
+    [ metavar "SUBDOMAIN"
+    , help    "Optional subdomain -- autogeneratd if omitted"
+
+    , value   ""
+
+    , long    "subdomain"
+    , short   's'
+    ]
+
+  return App.Init.Options
+    { appDir
+    , buildDir  = mayFilePath optFilePath
+    , subdomain = maySubdomain optSubdomain
```

Comment:
Annoying, but need to do down here because the parser is only an applicative, not a monad.